README.rst

```diff
@@ -1422,6 +1463,74 @@ Example usage:
         '*png',
     ]
 
+``OPENWISP_HTML_EMAIL``
+^^^^^^^^^^^^^^^^^^^^^^^
+
++---------+----------+
+| type    | ``bool`` |
++---------+----------+
+| default | ``True`` |
++---------+----------+
+
+If ``True``, attaches markdown rendered HTML of message in email.
+If ``False``, HTML rendering of message will be disabled and a plain
+text email is sent.
+
+``OPENWISP_EMAIL_TEMPLATE``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
++---------+----------------------------------------+
+| type    | ``str``                                |
++---------+----------------------------------------+
+| default | ``openwisp_utils/email_template.html`` |
++---------+----------------------------------------+
+
+This setting takes the path to the template for email. Thus, making it possible to
+customize email. You can either extend the default email template or write your own
+email template from scratch. An example of extending default email template to customize styling is
+shown below.
+
+.. code-block:: django
+
+    {% extends 'openwisp_utils/email_template.html' %}
+    {% block styles %}
+    {{ block.super }}
+    <style>
+      .background {
+        height: 100%;
+        background: linear-gradient(to bottom, #8ccbbe 50%, #3797a4 50%);
+        background-repeat: no-repeat;
+        background-attachment: fixed;
+        padding: 50px;
+      }
+
+      .mail-header {
+        background-color: #3797a4;
+        color: white;
+      }
+    </style>
+    {% endblock styles %}
+
+Similarly, you can customize the HTML of the template by overriding the ``body`` block.
+See `email_template.html <https://github.com/openwisp/openwisp-utils/blob/
+master/openwisp_utils/admin_theme/templates/openwisp_utils/email_template.html>`_
+for reference implementation.
+
+``OPENWISP_EMAIL_LOGO``
+^^^^^^^^^^^^^^^^^^^^^^^
+
++---------+-------------------------------------------------------------------------------------+
+| type    | ``str``                                                                             |
++---------+-------------------------------------------------------------------------------------+
+| default | `OpenWISP logo <https://raw.githubusercontent.com/openwisp/openwisp-utils/master/ \ |
+|         | openwisp_utils/static/openwisp-utils/images/openwisp-logo.png>`_                    |
++---------+-------------------------------------------------------------------------------------+
+
+This setting takes the URL of the logo to be displayed on email.
```

Comment:
This setting allows to change the logo which is displayed in the HTML version of the email.